app/src/main/java/org/wikipedia/createaccount/CreateAccountActivity.kt

```diff
@@ -207,13 +225,15 @@ class CreateAccountActivity : BaseActivity() {
         L.w("Account creation failed with result $message")
     }
 
-    private fun doCreateAccount(token: String) {
+    private fun doCreateAccount(token: String, hCaptchaToken: String? = null) {
         showProgressBar(true)
         val email = getText(binding.createAccountEmail).ifEmpty { null }
         val password = getText(binding.createAccountPasswordInput)
         val repeat = getText(binding.createAccountPasswordRepeat)
         val userName = getText(binding.createAccountUsername)
-        viewModel.doCreateAccount(token, captchaHandler.captchaId().toString(), captchaHandler.captchaWord().toString(), userName, password, repeat, email)
+        viewModel.doCreateAccount(token, captchaHandler.captchaId(),
+            if (hCaptchaToken.isNullOrEmpty()) captchaHandler.captchaWord() else hCaptchaToken,
+            userName, password, repeat, email)
```

Comment:
Can we include parameter names to improve readability?
```
viewModel.doCreateAccount(
    token = token, 
    captchaId = captchaHandler.captchaId(),
    captchaWord = if (hCaptchaToken.isNullOrEmpty()) captchaHandler.captchaWord() else hCaptchaToken,
    userName = userName, 
    password = password,
    repeat = repeat, 
    email = email
)
```

Reply:
Sure!